src/Shopsys/ShopBundle/Resources/styles/front/common/components/list/products.less

```diff
@@ -124,30 +124,12 @@
             .is-no-flex & {
                 display: block;
             }
-        }
-
-        &__block {
-            display: flex;
-            flex: 1;
-            flex-direction: column;
```

Comment:
This change was not made in the original commits, please check this.

Reply:
That seems like a glitch in diffing on Github. The compare of the two commits (my reverting commit c4d698303b15ffddcfe250fc5a0ebd4634307a27 and last pre-design-push `master` commit 2eaeb9f6fdea2d179f543eb2a6ce7d02de68a85d) is empty: https://github.com/shopsys/demoshop/compare/2eaeb9f6fdea2d179f543eb2a6ce7d02de68a85d...c4d698303b15ffddcfe250fc5a0ebd4634307a27.

If you don't trust the GitHub compare (and I guess you shouldn't, taking in mind my comment starts with *That seems like a glitch in diffing on Github...*), take a look at the raw file on those commits:
- https://raw.githubusercontent.com/shopsys/demoshop/2eaeb9f6fdea2d179f543eb2a6ce7d02de68a85d/src/Shopsys/ShopBundle/Resources/styles/front/common/components/list/products.less (before)
- https://raw.githubusercontent.com/shopsys/demoshop/c4d698303b15ffddcfe250fc5a0ebd4634307a27/src/Shopsys/ShopBundle/Resources/styles/front/common/components/list/products.less (after)